src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/Et3ResponseController.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.controllers;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.et.common.model.ccd.CCDCallbackResponse;
+import uk.gov.hmcts.et.common.model.ccd.CCDRequest;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.ethos.replacement.docmosis.helpers.Et3ResponseHelper;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.VerifyTokenService;
+
+import static org.springframework.http.HttpStatus.FORBIDDEN;
+import static org.springframework.util.MimeTypeUtils.APPLICATION_JSON_VALUE;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.YES;
+import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.CallbackRespHelper.getCallbackRespEntityNoErrors;
+
+
+/**
+ * REST controller for the ET3 Response pages, formats data appropriately for rendering on the front end.
+ */
+@Slf4j
+@RequestMapping("/et3Response")
+@RestController
+@RequiredArgsConstructor
+public class Et3ResponseController {
+
+    private static final String INVALID_TOKEN = "Invalid Token {}";
+    private final VerifyTokenService verifyTokenService;
+
+    /**
+     * Called for the start of the ET3 Response Form event.
```

Comment:
add why you are setting the variables to Yes. no context as to why they are set to yes